web/ui/react-app/src/vendor/flot/jquery.flot.heatmap.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * Inspired by a similar feature in VictoriaMetrics.
+ * See https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3384 for more details.
+ * Developed by VictoriaMetrics team.
+ */
```

Comment:
Added the MIT license to `jquery.flot.heatmap.js`

Reply:
Now I'm a bit confused because VictoriaMetrics is Apache 2 licensed and not MIT.

I first thought that maybe the part of the codebase where this file is copied from is MIT licensed. But I couldn't even find a file that is the same (or at least similar) to this one.

Where did you copy this file from?

Or alternatively, if you created this file just for this PR, then we don't really need to put it in a vendor directory (as it isn't vendored), and we can use the usual Apache 2 license and don't need a copyright header.

Reply:
This file was created for this PR. I saw that all plugins for flot were in this folder, so I placed it there. I can create a new folder, for example `flot/plugins`, and move it there, or you can suggest a better place for it.

Reply:
@Nexucis might also be able to help here.